src/frontend/dash/renderCircles.ts

```diff
@@ -3,18 +3,23 @@ import type { MemberCircle } from "./circlePacker"
 
 const membersDiv = document.getElementById('members')
 
+
 const BUBBLE_COLORS = ['rgba(35,132,198,.5)', 'rgba(255,214,0,.5)', 'rgba(241,93,34,.5)', 'rgba(108,157,204,.5)']
 function renderCircle(circle: MemberCircle) {
     const { maxX, maxY, minX, minY } = getBounds()
-    const maxLength = Math.max(maxX - minX, maxY - minY)
+
+    const widthMultiplier = window.outerWidth * 0.4750078125 / membersDiv.clientWidth
```

Comment:
Are these just for making the circles smaller? Also, does this code make the circles bigger again once chiefDelphi is hidden?

Reply:
membersDiv will expand when cheif delphi is not displayed. the width and height multipliers track that.

Reply:
Interesting, why not just have it take up the whole screen if chief delphi isn't there?